Remotion/Web/UnitTests/Core/ExecutionEngine/WxeContextTest.cs

```diff
@@ -39,9 +41,12 @@ public class WxeContextTest
   private string _functionTypeName;
   private string _resource;
 
+  private ServiceLocatorScope _serviceLocatorScope;
+
   [SetUp]
   public virtual void SetUp ()
   {
+    _serviceLocatorScope = WebConfigurationFactory.GetBasicWxeUrlSettings();
```

Comment:
I'm not a fan of returning a fully formed ServiceLocatorScope from this Factory. THe name and the intention do not match.

Reply:
As this factory will only contain these methods soon anyways, we have free reign over how we name them and the class :)